docs/REST/authentication/index.md

```diff
@@ -0,0 +1,11 @@
+﻿# Authentication
+
+All HTTP requests must be authenticated. The following authentication schemes are supported.
+
+## HMAC
+
+[HMAC authentication](./authentication/hmac.md) uses a randomly generated secret to sign request payloads. Details on how requests using this authentication method are authorized can be found in the [HMAC authorization](../authorization/hmac.md) section.
```

Comment:
>./authentication/hmac.md [](start = 22, length = 24)

This link doesn't work - I'm seeing it resolve to authentication/authentication/hmac.md

Reply:
Yes it was broken. Just fixed it, thanks!